GCMessengerSDKSample/app/build.gradle

```diff
@@ -51,9 +51,9 @@ android {
 dependencies {
 
     //region Genesys
-    implementation("com.genesys.cloud:core:1.10.2.rc1")
-    implementation("com.genesys.cloud:chatintegration:1.10.2.rc1")
-    implementation("com.genesys.cloud:ui:1.10.2.rc1")
+    implementation("com.genesys.cloud:core:1.10.2.rc2")
```

Comment:
Please increase the versionCode too.

Reply:
This should be 56. 55 is my current release. We do need a solution for this :D

Reply:
@SzaszaGenesys , should we really increase `versionCode` when increase `rc` version? 

If so I can propose the following way of `versionCode` construction:

```
    def projectVersion = "1.10.2.rc"
    def parts = projectVersion.split('\\.')
    def major = parts[0].toInteger()
    def minor = parts[1].toInteger()
    def patch = parts[2].toInteger()
    
      def versionCode = (major * 1000000) + (minor * 10000) + (patch * 100)
    
    if (parts.length > 3) {
        def preRelease = parts[3].replaceAll('rc', '').toInteger()
        versionCode += preRelease
    }
```

"1.10.2.rc2" becomes 1100202
"1.10.2" becomes 1100200
"2.0.0" becomes 2000000
"0.9.5.rc1" becomes 9501

Reply:
@mobichel Sounds good. I guess you can create a ticket for it.